beetsplug/acoustic.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+
+
+from beets import plugins, ui
+import requests
+
+ACOUSTIC_URL = "http://acousticbrainz.org/"
+LEVEL = "/high-level"
+PLUGIN_DESCRIPTION = "Fetch metadata from AcousticBrainz"
+
+
+class AcousticPlugin(plugins.BeetsPlugin):
+    def __init__(self):
+        super(AcousticPlugin, self).__init__()
+
+    def commands(self):
+        cmd = ui.Subcommand('acoustic', help=PLUGIN_DESCRIPTION)
```

Comment:
The usual way to format long lines like this is to wrap inside the parentheses. For example:

```py
cmd = ui.Subcommand('acoustic',
                    help='...')
```

or:

```py
cmd = ui.Subcommand(
    'acoustic',
    help='...',
)
```

rather than using a constant.